### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by DavidIanSteele**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
 # Test
 
 A small test repo to get up to speed with new workflow
+
+More Info
+=========
```
Comment (DavidIanSteele) [inline README.md]: The wrong kind of header?

Comment (DavidIanSteele) [inline README.md]: !resolved

Comment (adeelkazmi) [inline README.md]: I don't like this !fix

Comment (adeelkazmi) [inline README.md]: !fixed

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # Test
 
 A small test repo to get up to speed with new workflow
+
+# More Info #
```
Comment (adeelkazmi) [inline README.md]: !fix

Comment (adeelkazmi) [inline README.md]: !fixed

**Pull request merged**